app/main/views/questionnaire.py

```diff
@@ -276,6 +269,24 @@ def post_household_composition(eq_id, form_type, collection_id, group_id):
     return redirect(location_url(eq_id, form_type, collection_id, next_location))
 
 
+@questionnaire_blueprint.route('<group_id>/<int:group_instance>/permanent-or-family-home', methods=["POST"])
+@login_required
+def post_everyone_at_address_confirmation(eq_id, form_type, collection_id, group_id, group_instance):
+    _remove_repeating_on_household_answers(get_answer_store(current_user), group_id)
```

Comment:
Do we only want to do this if the answer is no?
If i navigate backwards and then forwards without changing the answer then the values are deleted.